index.js

```diff
@@ -1,58 +1,38 @@
 const Assert = require('assert')
-const Child = require('child_process')
-const FS = require('fs')
-const Parallel = require('run-parallel')
 const Path = require('path')
+const Parallel = require('run-parallel')
+const { command, project } = require('./lib')
 
 // get details about package and environment
-module.exports = (options, done) => {
-  if (typeof options === 'function') {
-    done = options
-    options = {}
+module.exports = (commands, done) => {
+  if (typeof commands === 'function') {
+    done = commands
+    commands = {}
   }
 
-  Assert.equal(typeof options, 'object', 'Options must be an object')
-  Assert.equal(typeof done, 'function', 'Must pass in a callback function')
+  Assert(isObject(commands), '`commands` must be an object')
+  Assert(isFunction(done), 'Must pass in a callback function')
 
   Parallel([
-    packageDetails,
     (next) => process.nextTick(next, null, { env: process.env.NODE_ENV }),
-    (next) => {
-      const cmds = Object.assign({ node: 'node -v', npm: 'npm -v' }, options)
-
-      commandDetails(cmds, next)
-    }
+    (next) => project(Path.dirname(require.main.filename), next),
+    (next) => commandDetails(commands, next)
   ], (err, results) => {
-    if (err) done(err)
-    else done(null, Object.assign({}, ...results))
+    done(err, Object.assign({}, ...results))
```

Comment:
Actually `err` never exists, all the `tasks` yield an `{}` when things go wrong. So, mostly this is just a simpler way to avoid adding `// eslint-disable-line handle-callback-error` and still get 100% coverage.